Foundation/NSURLSession/http/EasyHandle.swift

```diff
@@ -172,18 +169,13 @@ extension _EasyHandle {
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionPROTOCOLS, protocols).asError()
         try! CFURLSession_easy_setopt_long(rawHandle, CFURLSessionOptionREDIR_PROTOCOLS, protocols).asError()
         #if os(Android)
-            // See https://curl.haxx.se/docs/sslcerts.html
-            // For SSL to work you need "cacert.pem" to be accessable
-            // at the path pointed to by the URLSessionCAInfo env var.
-            // Downloadable here: https://curl.haxx.se/ca/cacert.pem
-            if let caInfo = _EasyHandle._CAInfoFile  {
-                if String(cString: caInfo) == "UNSAFE_SSL_NOVERIFY" {
-                    try! CFURLSession_easy_setopt_int(rawHandle, CFURLSessionOptionSSL_VERIFYPEER, 0).asError()
-                }
-                else {
-                    try! CFURLSession_easy_setopt_ptr(rawHandle, CFURLSessionOptionCAINFO, caInfo).asError()
-                }
-            }
+        if URLSession.sslUnsafeBypassPeerVerify {
+            try! CFURLSession_easy_setopt_int(rawHandle, CFURLSessionOptionSSL_VERIFYPEER, 0).asError()
+        }
+        else if let caInfo = URLSession._sslCertificateAuthorityFile {
+            try! CFURLSession_easy_setopt_int(rawHandle, CFURLSessionOptionSSL_VERIFYPEER, 1).asError()
+            try! CFURLSession_easy_setopt_ptr(rawHandle, CFURLSessionOptionCAINFO, caInfo).asError()
+        }
```

Comment:
Else...? That is, what happens if the user wants verification but has a `nil` value for `_sslCertificateAuthorityFile`?